tests/test_external_search.py

```diff
@@ -890,6 +890,14 @@ def test_query_works(self):
         )
         expect([self.pride], "pride and prejudice", f)
 
+        # Here, a different data source is excluded, and it shows up.
+        f = Filter(
+            excluded_audiobook_data_sources=[
+                DataSource.lookup(self._db, DataSource.BIBLIOTHECA)
+            ]
+        )
+        expect([self.pride, self.pride_audio], "pride and prejudice", f)
+
```

Comment:
This test isn't related to any of the code in this branch; I added it while debugging a problem that turned out to be unrelated, and keeping it seems like a good idea.